### PR TITLE
Getting started update

### PR DESCRIPTION
**Pull request by msimecek**
Based on the feedback from yesterdays HOL. Feel free to add more.

### CODE REVIEW

**Comment by msimecek** [comment]
The elephant in the room is still: should we include both Bash and PowerShell snippets?

**Comment by heoelri** [comment]
> The elephant in the room is still: should we include both Bash and PowerShell snippets?

I'd vote for no. The guide should be as lightweight and straightforward as possible. 

We'd some trouble with the Service Principals as well. It might make sense to highlight that as well.

File: docs/reference-implementation/Getting-Started.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ az pipelines create --name "Azure.AlwaysOn E2E Release" --description "Azure.Alw
 
 All pipelines require an Azure DevOps service connection to access the target Azure Subscription where the resources are deployed. These service connections use Service Principals to access Azure which can be configured automatically, when proper access is given, or manually in Azure DevOps by providing a pre-created Azure Service Principal with the required permissions.
 
-We need to create an AAD Service Principal with **Subscription-level Owner permissions**. We need owner permission as the pipeline will need to create various role assignments.
+> **Important:** You need to create an AAD Service Principal with **Subscription-level Owner permissions**. We need owner permission on the whole subscription as the pipeline will need to create various role assignments.
```
Comment (heoelri) [inline docs/reference-implementation/Getting-Started.md:164]:
```suggestion
> **Important!** The AAD Service Principal needs **subscription-level owner permissions** as the pipeline will create various role assignments.
```

Comment (msimecek) [reply docs/reference-implementation/Getting-Started.md:164]: Ok, let's increase the urgency a bit :D

Comment (heoelri) [reply docs/reference-implementation/Getting-Started.md]: **Important!** for consistency - this is the notation that's used in 3 other cases in this document.

And the second part for easier readability - its more likely that someone reads a single sentence than 2.. up to you. Just an idea.

**Review by heoelri — APPROVED** [review]
LGTM

File: docs/reference-implementation/Getting-Started.md (hunk 1 above)
Comment (msimecek) [reply docs/reference-implementation/Getting-Started.md]: I like it! More concise and clear language always wins in my view.

**Comment by sebader** [comment]
> The elephant in the room is still: should we include both Bash and PowerShell snippets?

you mean both? I'm almost for dropping CLI command completely from the guide

**Comment by msimecek** [comment]
> > The elephant in the room is still: should we include both Bash and PowerShell snippets?
> 
> you mean both? I'm almost for dropping CLI command completely from the guide

I used it when going through the guide and it's much more efficient and less error-prone than clicking in the portal. So I'd definitely keep the commands there.

File: docs/reference-implementation/Getting-Started.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ The following tools and applications must be installed on the client machine use
 
 - Install [Azure DevOps CLI](https://docs.microsoft.com/azure/devops/cli/?view=azure-devops)
```
Comment (sebader) [inline docs/reference-implementation/Getting-Started.md:21]: I would mark this as optional - based on whether somebody wants to follow the CLI path or not

File: docs/reference-implementation/Getting-Started.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ The following tools and applications must be installed on the client machine use
...
 
-- Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.1).
+- Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell?view=powershell-7.1) (on Windows, Linux or macOS).
```
Comment (sebader) [inline docs/reference-implementation/Getting-Started.md:23]: same as above

File: docs/reference-implementation/Getting-Started.md [hunk 4]
```diff
@@ -201,7 +201,9 @@ Our AlwaysOn reference implementation knows three different environments: prod,
 
 > If you only created one Service Principal above, you only need to create one Service Connection for now.
 
-These service connections can be created in the Azure DevOps Portal or via the `az devops` CLI. Create them using either one of these two methods.
+> **Important!** Since these connection names are used in pipelines, use them exactly as specified above. If you change the name of the service connection, you have to also change it in pipeline YAML.
```
Comment (sebader) [inline docs/reference-implementation/Getting-Started.md:204]: In the lines above, can we make it a bit more clear to use exactly these names?

**Review by sebader — CHANGES_REQUESTED** [review]

File: docs/reference-implementation/Getting-Started.md (hunk 4 above)
Comment (msimecek) [reply docs/reference-implementation/Getting-Started.md]: How about now?

File: docs/reference-implementation/Getting-Started.md [hunk 5]
```diff
@@ -14,13 +14,14 @@ This guide describes the end-to-end process for setting up all pre-requisites an
...
 ## Pre-requisites
 
-The following tools and applications must be installed on the client machine used to deploy AlwaysOn reference implementation:
+The following must be installed on the client machine used to deploy AlwaysOn reference implementation:
 
-- Install [Azure CLI](https://docs.microsoft.com/cli/azure/service-page/azure%20cli?view=azure-cli-latest)
+- [Azure CLI](https://docs.microsoft.com/cli/azure/service-page/azure%20cli?view=azure-cli-latest)
 
-- Install [Azure DevOps CLI](https://docs.microsoft.com/azure/devops/cli/?view=azure-devops)
+This guide offers two paths: UI or script. If you prefer scripting, make sure you have these additional tools installed:
```
Comment (sebader) [inline docs/reference-implementation/Getting-Started.md:21]:
```suggestion
This guide offers two paths: Using Azure DevOps Portal or script-based via Azure DevOps CLI. If you prefer scripting, make sure you have these additional tools installed:
```

**Review by sebader — CHANGES_REQUESTED** [review]

File: docs/reference-implementation/Getting-Started.md (hunk 4 above)
Comment (sebader) [reply docs/reference-implementation/Getting-Started.md]: nice! thanks

**Review by sebader — APPROVED** [review]
thanks!

**Comment by msimecek** [comment]
https://github.com/Azure/AlwaysOn-Foundational-Connected/pull/101

**Pull request merged**